gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApiResource.java

```diff
@@ -277,6 +302,45 @@ public Response exportApiDefinition(@Context HttpHeaders headers, @PathParam("ap
             .build();
     }
 
+    @POST
+    @Path("/_duplicate")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Permissions(
```

Comment:
To verify, but I think this list of permissions works like a "OR". IT means you need at least one of these two permissions. But in the openapi doc, it's written "AND".
I think you will have to change the doc or change the permissions check here.

Reply:
Good catch. I've copied/pasted from v1 rest api. I'm updating the check to ensure the AND